### PR TITLE
Fix clearing TextInput in google chrome

### PR DESCRIPTION
**Pull request by psegedy**

### CODE REVIEW

**Review by john-dupuy — APPROVED** [review]

**Review by bsquizz — APPROVED** [review]

**Comment by bsquizz** [comment]
FYI @mshriver @quarckster  -- can you guys take a look?

File: src/widgetastic/widget/input.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def fill(self, value):
             return False
         # Clear and type everything
         self.browser.click(self)
-        self.browser.clear(self)
+        self.browser.execute_script("arguments[0].value = '';", self)
```
Comment (quarckster) [inline src/widgetastic/widget/input.py:63]: It should be done in `browser.clear` method as a case for chrome and a specific version.

**Review by quarckster — CHANGES_REQUESTED** [review]

File: src/widgetastic/widget/input.py (hunk 1 above)
Comment (digitronik) [reply src/widgetastic/widget/input.py:63]: If it's specific to a browser or some location better to go with a local widget (overwrite fill method).
As its core lib; I think if we fallow the selenium default call (execute `clearElement` cmd) will be a good option.

Comment (psegedy) [reply src/widgetastic/widget/input.py]: done

**Review by quarckster — APPROVED** [review]

**Pull request merged**